frontend/src/component/project/Project/Project.tsx

```diff
@@ -180,7 +180,7 @@ export const Project = () => {
         },
         {
             title: 'Project settings',
-            path: `${basePath}/settings${isOss() ? '/environments' : ''}`,
+            path: `${basePath}/settings`,
```

Comment:
This was causing the issue in https://github.com/Unleash/unleash/issues/8618, appending environments to route, since previously OSS only saw environments.